src/EnergyPlus/ZonePlenum.cc

```diff
@@ -1138,22 +1161,19 @@ namespace ZonePlenum {
 				}
 				Node( ZoneNode ).CO2 = Node( OutletNode ).CO2;
 			} else {
-				Node( OutletNode ).CO2 = Node( InletNode ).CO2;
-				Node( ZoneNode ).CO2 = Node( InletNode ).CO2;
+				Node( OutletNode ).CO2 = Node( ZoneNode ).CO2;
```

Comment:
This is the functional change to fix this PR, right?  It previously just moved the inlet node info to the outlet and zone nodes.  Now it moves the zone node to the outlet node and that is all?  So where the is ZoneNode updated now?  I see calls above to set the InducedNode conditions; is that it?